### ISSUE TITLE
Term definition cross-links in understanding don't work

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#1541]
Taking https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html#dfn-functionality as an example, the definition there has a cross-reference to the term "processes". But the link there is broken - it points to https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html#dfn-process which does not exist (as the term was not pulled into the generated understanding document). Not sure this can be solved with any XSLT magic ... as this almost requires some sort of recursive "go fetch, and add, any term glossary definitions ... and then, go check THOSE for any links to other terms"

Maybe they should be automagically rewritten to point back to the main spec's glossary to at least make sure they work

**Comment by jake-abma** [comment]
refer https://github.com/w3c/wcag/issues/1525

think if we take a better look we find much more...

**Comment by patrickhlauke** [comment]
yes, it's basically any time a key term, which *is* pulled in automatically if it's present in the normative SC text, contains any other key term/definition which is *not* present in the normative SC text.

(might be worth keeping this issue open as it's worded a bit more generically, and closing #1525 ?)

**Comment by alastc** [comment]
@michael-n-cooper could you take a look, is there an automated (template) solution?
